doc/basics/compilation.md

```diff
@@ -9,7 +9,7 @@ All the dependencies are automatically fetched and compiled at build time.
  * [GuiBase] Qt Core, Qt Widgets and Qt OpenGL v5.5+ (5.10 recommended)
  * stb_image
 
-See @ref develbuildchain for technical details on dependencies management.
+See @ref develbuildchain for technical details on dependencies management and how to provide locally installed dependencies.
```

Comment:
Hum good idea... 
But, ... can we keep that for another PR ? I mean the current one is large enough, and what you suggest is to add another functionality, so can be done after.

Reply:
I think we can do that on this PR. Just 1 file more (and a small documentation adds). I'll try to do that this morning.

Reply:
Well, it's not so hard, but then we will have to maintain two files when doing update of dependencies ...

Reply:
I've written a `CMakeLists.txt` that just include the one in Core, Engine and IO. 
As far as I know, we can update the dependencies of Radium without changing something in this file (but only in external/Core, external/Engine and external/IO), so, this will not change the work to do when updating a dependency.
This file will be alone in one commit, so that, if you don't agree, we can remove this commit.
I'm testing for now and I'll push very quickly.

Reply:
As I said I prefer that we add this in a dedicated PR.

Reply:
> As I said I prefer that we add this in a dedicated PR.

Give a look on the commit 4d49506878bf6f337a440e8b8bf518484751805d and feel free to remove it if you think it is too intrusive. 
I do not think so but ...
I tried it with all or only some of the dependencies, everything goes fine.